extensions/kafka-client/deployment/src/main/resources/dev-ui/qwc-kafka-add-message.js

```diff
@@ -0,0 +1,249 @@
+import { LitElement, html, css} from 'lit'; 
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/form-layout';
+import '@vaadin/text-field';
+import '@vaadin/combo-box';
+import '@vaadin/text-area';
+import '@vaadin/button';
+
+/**
+ * This component shows the Add Message screen
+ */
+export class QwcKafkaAddMessage extends LitElement { 
+    
+    static styles = css`
+
+        :root {
+            display: flex; 
+            flex-direction: column; 
+        }
+
+    `;
+
+    static properties = {
+        partitionsCount: {type: Number},
+        topicName: {type: String},
+        extensionName: {type: String}, // TODO: Add 'pane' concept in router to register internal extension pages.
+        _targetPartitions: {state: false},
+        _types: {state: false},
+        _newMessage: {state: true},
+        _newMessageHeaders: {state: true, type: Array},
+    };
+
+    constructor() { 
+        super();
+        this.partitionsCount = 0;
+        this.topicName = null;
+        this._targetPartitions = [];
+        this._types = [];
+        this._reset();
+        this.responsiveSteps = [
+            { minWidth: 0, columns: 1 },
+            { minWidth: '320px', columns: 2 },
+        ];
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+        this.jsonRpc = new JsonRpc(this.extensionName);
+
+        this._targetPartitions.push({name: "Any", value: "any"});
+        for (var i = 0; i < this.partitionsCount; i++) {
+            this._targetPartitions.push({name: i.toString(), value: i});
+        }
+
+        this._types.push({name: "Text", value: "text"});
+        this._types.push({name: "None (Tombstone)", value: "none"});
+        this._types.push({name: "JSON", value: "json"});
+        this._types.push({name: "Binary", value: "binary"});
+    }
+
+    _reset(){
+        this._newMessage = new Object();
+        this._newMessage.partition = 'any';
+        this._newMessage.type = 'text';
+        this._newMessage.key = null;
```

Comment:
We would be better if we put empty string to key and value here.

Reply:
Is the above not the default ?